apps/builder/app/builder/shared/commands.ts

```diff
@@ -310,6 +311,20 @@ export const { emitCommand, subscribeCommands } = createCommandsEmitter({
         $breakpointsMenuView.set("initial");
       },
     },
+    {
+      name: "openPublishDialog",
+      defaultHotkeys: ["shift+P"],
```

Comment:
I would go with either shift+p or P

Reply:
shift+p is basically shift+P because shift makes it big

Reply:
At least that's how webflow comunicates it as well